modules/KIWILinuxRC.sh

```diff
@@ -23,6 +23,7 @@ export ELOG_FILE=/var/log/boot.kiwi
 export TRANSFER_ERRORS_FILE=/tmp/transfer.errors
 export UFONT=/usr/share/fbiterm/fonts/b16.pcf.gz
 export CONSOLE_FONT=/usr/share/kbd/consolefonts/default8x16.gz
+export HYBRID_PERSISTENT_FILENAME="!Live OS's persistent storage.fs"
```

Comment:
That's a strange name for linux users :-) I guess you use that to make it more clear on the Windows side when Windows provides a view on the iso ? if so a comment would be nice.